### ISSUE TITLE
508-defect-1 ❗️ Launchblocker [COGNITION, SCREEN READER]: Checkboxes MUST have unique and accessible labels

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#24897]
# [508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## User Story or Problem Statement

While running axe checks on the Notice of Disagreement review & submit page, the frontend developer discovered that the form system was not adding unique (page indexed) IDs to the checkbox groups on the area of disagreement repeated pages.

## Details

 When more than one area of disagreement page is edited on the review & submit page, a (1) unique ID and (2) form field must not have multiple label elements errors show up

## Acceptance Criteria

- [ ]  All instances of the error are fixed
- [ ]  Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Environment

- https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/review-and-submit

## Steps to Recreate

1. Go to https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/introduction
2. Log in to one of the HLR test users - https://dsva.slack.com/archives/C5AGLBNRK/p1601069669025200?thread_ts=1601069567.024300&cid=C5AGLBNRK
3. Navigate through the form (jumping directly to the review & submit page doesn't work properly) and fill out required sections 
4. On the review & submit page open the "Issues for review" accordion
5. Edit 2 or more issues (above the "Opt in" section.
6. Run axe DevTools and note the Id and form label errors similar to the screenshot 

## Proposed Solution (if known)

Needs discovery - adding an ID to array label IDs

## Screenshots or Trace Logs

![Screen Shot 2021-05-18 at 3 09 12 PM](https://user-images.githubusercontent.com/136959/118718648-1ad17980-b7ed-11eb-9efd-a22eb1e5d7f1.png)

**Comment by joshkimux** [comment]
@Mottie nice catch!! I didn't notice this on my first run through
<img width="1437" alt="Screen Shot 2021-05-24 at 8 26 04 AM" src="https://user-images.githubusercontent.com/14154792/119347610-bb022500-bc69-11eb-9c36-77d4fc41c627.png">

**Comment by Mottie** [comment]
The fix has been merged into master. @joshkimux I'm not sure what you meant by using arrow keys to navigate the checkboxes? I thought that was only possible with radio buttons?

**Comment by joshkimux** [comment]
@Mottie you're right-- ignore what I said earlier, my brain isn't functioning 😆
